lib/options/format.js

```diff
@@ -28,7 +28,8 @@ var DEFAULTS = {
   indentBy: 0,
   indentWith: IndentWith.Space,
   spaces: spaces(false),
-  wrapAt: false
+  wrapAt: false,
+  addSemicolonAfterLastPropertyInPropertyBlock: false
```

Comment:
I think we could name it `semicolonAfterLastProperty` as semicolons are not added anywhere else.